static/js/hotkey.js

```diff
@@ -407,6 +422,97 @@ exports.process_hotkey = function (e, hotkey) {
             return exports.process_escape_key(e);
     }
 
+    var draft_arrow = drafts.draft_model.get();
```

Comment:
We should just have all this in an if statement `if ($("#draft_overlay").hasClass("show"))`